cli/server/server.go

```diff
@@ -119,26 +120,35 @@ func getConfigFromContext(ctx *cli.Context) (config.Config, error) {
 // handleLoggingParams reads logging parameters.
 // If user selected debug level -- function enables it.
 // If logPath is configured -- function creates dir and file for logging.
-func handleLoggingParams(ctx *cli.Context, cfg config.ApplicationConfiguration) error {
+func handleLoggingParams(ctx *cli.Context, cfg config.ApplicationConfiguration) (*zap.Logger, error) {
+	level := zapcore.InfoLevel
 	if ctx.Bool("debug") {
-		log.SetLevel(log.DebugLevel)
+		level = zapcore.DebugLevel
 	}
 
+	cc := zap.NewDevelopmentConfig()
```

Comment:
Why you’re used development logger?

Reply:
I think we're better using production config by default. If you need something for development --- either keep a local patch for it or make it configurable (effectively moving this patch from code patch to config patch).

Reply:
fix that